cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -610,6 +610,19 @@ var __AdyenHelper: Object = {
         return jsonObject;
     },
 
+    /**
+     * Checks if payment method is blocked
+     */
+    isMethodTypeBlocked : function(methodType) {
+        if (methodType.indexOf('bcmc') !== -1 ||
+            (methodType.indexOf('wechatpay') !== -1 && methodType.indexOf('wechatpayWeb') === -1) ||
+            methodType == "scheme" || methodType == "cup" || methodType == "applepay") {
```

Comment:
are we not missing bcmc_mobile_QR ?

Reply:
This will also be included with "methodType.indexOf('bcmc')"